spec/controllers/crud_spec.rb

```diff
@@ -0,0 +1,40 @@
+require_relative '../spec_helper'
+
+describe 'index controller' do
+  it 'should process the get /sealions route' do
+    get '/sealions'
+    expect(last_response).to be_ok
+    expect(last_response.body).to include('The best')
+  end
+
+  it 'should process the post /post_something route' do
+    # post '/post_something', :params => {info: mypost}
```

Comment:
I think this should work. Perhaps you need to `puts` `params` in the controllers?